src/utils/transactions.ts

```diff
@@ -49,6 +49,12 @@ export class SafeTransaction {
   }
 }
 
+/**
```

Comment:
the baseOfsset in line 40 should probably be `signers.length * 65` as we need the offset in bytes not in chars.